content/terms/self-hosted.md

```diff
@@ -322,17 +322,17 @@ EXCEPT WHERE PROHIBITED BY LAW OR FEES OWED EXCEED THE BELOW LIMIT, NEITHER PART
   <i>**Short version**: Don’t use our site or software if you’re located in an embargoed country.</i>
 </span>
 
-The Software and the underlying information and technology may not be downloaded or otherwise exported or re-exported (a) into (or to a national or resident of) any country to which the U.S. has embargoed goods; or (b) to anyone on the U.S. Treasury Department’s list of Specially Designated Nationals or the U.S. Commerce Department’s Table of Deny Orders. By downloading or using the Software, you are agreeing to the foregoing and you represent and warrant that you are not located in, under the control of, or a national or resident of any such country or on any such list and you agree to comply with all export laws and other applicable laws.
+<br>The Software and the underlying information and technology may not be downloaded or otherwise exported or re-exported (a) into (or to a national or resident of) any country to which the U.S. has embargoed goods; or (b) to anyone on the U.S. Treasury Department’s list of Specially Designated Nationals or the U.S. Commerce Department’s Table of Deny Orders. By downloading or using the Software, you are agreeing to the foregoing and you represent and warrant that you are not located in, under the control of, or a national or resident of any such country or on any such list and you agree to comply with all export laws and other applicable laws.
 
 <span className="tw-text-blurple-400">
   ### **13. Governing law; arbitration; and class action/jury trial waiver**
 </span>
 
 <span className="tw-text-gray-400">
-  <i>**Short version**: We’re based in California, so the agreement is governed by California law, and all disputes must be brought there. If you have a claim against us, you’ll work with us to arbitrate it on an individual basis instead of via class action or jury trial.</i>
+  <i>**Short version**: If you're based in California like we are, the agreement is governed by California law, and all disputes must be brought there; otherwise, the Agreement is governed by New York law and disputes must be brought in New York. If you have a claim against us, you’ll work with us to arbitrate it on an individual basis instead of via class action or jury trial.</i>
```

Comment:
Another tip, Instead of using HTML for rendering copy in bold or italics, wrap your text in two underscores for bold, and one asterisk for italics. (ie: `*__This is bold__ and this entire line is italicized*`